fast64_internal/oot/actor/properties.py

```diff
@@ -110,22 +164,193 @@ def draw_props(
 class OOTActorProperty(PropertyGroup):
     actorID: EnumProperty(name="Actor", items=ootData.actorData.ootEnumActorID, default="ACTOR_PLAYER")
     actorIDCustom: StringProperty(name="Actor ID", default="ACTOR_PLAYER")
+
+    # used for actors with the id "Custom"
     actorParam: StringProperty(name="Actor Parameter", default="0x0000")
     rotOverride: BoolProperty(name="Override Rotation", default=False)
-    rotOverrideX: StringProperty(name="Rot X", default="0")
-    rotOverrideY: StringProperty(name="Rot Y", default="0")
-    rotOverrideZ: StringProperty(name="Rot Z", default="0")
+    rotOverrideX: StringProperty(name="Rot X", default="0x0000")
+    rotOverrideY: StringProperty(name="Rot Y", default="0x0000")
+    rotOverrideZ: StringProperty(name="Rot Z", default="0x0000")
```

Comment:
I mean, sure, but how am I supposed to handle the getter function? you need to return something there, I'm not sure how to handle that

the setter should be doable though

Reply:
btw to clarify, the issue I'm having here is that I think you can't have a getter function that doesn't return anything, it will either raise an error or set a blank value which is not what we want here, for the setter you can just return the value from the parameters and it should be fine

also there's a scenario where I'm just completely stupid and forgot how this works

Reply:
I think I understand, the point of having a separate property is that its a way to get/set actor-specific properties? In that case it should be fine, but you should add a comment explaining the distinction between the duplicate properties.

Reply:
``rotOverride[X-Y-Z]`` is only used for the actor id "custom" to be able to write any value we want, I'm gonna rename those to ``rot_x_custom`` etc to make it clearer (and I'll add a comment to explain why it's required)